assets/prometheus-k8s/cluster-role.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/part-of: openshift-monitoring
     app.kubernetes.io/version: 2.32.1
   name: prometheus-k8s
-  namespace: openshift-monitoring
```

Comment:
This got removed due to https://github.com/openshift/cluster-monitoring-operator/pull/1554#discussion_r802525716